docs/sources/fundamentals/dashboards-overview/index.md

```diff
@@ -0,0 +1,88 @@
+
+The following image illustrates each of the gates, and the following sections describe each element, how it’s used, and why it’s necessary. The journey of a visualization begins at the data source and works its way through to a Grafana dashboard.
+
+{{< figure src="/media/docs/grafana/dashboards-overview/dashboard-component-architecture.png" max-width="750px" caption="Dashboard component architecture" >}}
+
+## Data Source
+
+A data source is any entity that contains data. A data source can be a SQL database, Grafana Loki, Grafana Mimir, or a JSON-based API. A data source can even be a flat CSV file. A dashboard visualization starts with the data source that contains the data you want to see.
+
+Understanding the differences between various data sources can be a challenge because each data source has a unique structure and requires different query methods. To simplify this complexity, Grafana Labs has developed plugins that unify these data sources into one coherent view.
+
+## Plugins
+
+A Grafana plugin is software that adds new capabilities to Grafana. They come in many types, but right now we will address _data source plugins_. The job of a Grafana data source plugin is to take a query you want answered, retrieve the  data from the data source, and reconcile the differences in data models using a unified data structure called a [data frame](https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/). The data coming into the plugin from the data source might be many different formats (such as JSON, rows and columns, or CSV), but when it leaves the plugin and moves through the rest of the gates toward a visualization, it is always data frames.
+
+At the time of this writing, there are 155 different kinds of data sources available to use with Grafana.  The most popular options come pre-installed installed and  ready to use. After you determine your needs, first check to see if a data source is already available. Grafana is expanding the list all the time, so if you don’t find what you need, you can [check the plugin catalog](https://grafana.com/grafana/plugins/?type=datasource) or even [build your own plugin](https://grafana.com/tutorials/build-a-data-source-plugin/).
+
+## Query
+
+Query enable you to reduce the data to just a specific dataset, so you aren't trying to visualize the entire thing. A query is an answer to a question you have about your system and operations. For example, if your company has an online presence, you might be interested in understanding how many customers are adding products to the shopping cart. To answer that question, you can write a query that  takes all of the access metrics for a shopping cart service, and aggregates them into how many users per second are accessing the service.
```

Comment:
```suggestion
Queries allow for the reduction of data to a specific dataset, providing a more manageable visualization. They help answer questions you have about system and operational processes. For instance, a company with an online store might want to determine the number of customers who add products to their shopping carts. This can be achieved through a query that aggregates access metrics for the shopping cart service, revealing the number of users accessing the service per second.
```